src/generator/ArtifactGenerator.js

```diff
@@ -336,58 +336,52 @@ class ArtifactGenerator {
       const generatedFullArtifactList = [];
 
       this.artifactData.artifactToGenerate.forEach((artifactInfo, index) => {
-        if (!artifactInfo.packaging) {
-          throw new Error(
-            `No packaging information for artifact number [${index}] from ${describeInput(
-              this.artifactData,
-            )}.`,
-          );
-        }
-
-        // TODO: manage repositories properly
-        this.artifactData.gitRepository = artifactInfo.gitRepository;
-        this.artifactData.repository = artifactInfo.repository;
-        artifactInfo.packaging.forEach((packagingInfo) => {
-          debug(
-            `Generating [${artifactInfo.programmingLanguage}] packaging for [${packagingInfo.packagingTool}]`,
-          );
+        if (artifactInfo.packaging) {
+          // TODO: Manage repositories properly.
```

Comment:
Can you create an associated ticket, and reference it in the todo, so that this doesn't just remain here unattended? Or if we are no longer sure what this is about let's just delete the TODO.

Reply:
Yeah, pretty sure I didn't add this comment (as it was missing a full-stop!), so I'm not sure what 'properly' means here myself.
So I'll just delete the TODO: 076b3cf